server/src/engineModel/EngineModel.ts

```diff
@@ -92,9 +93,7 @@ export class EngineModel {
                 this.completionProvider.addCompletionItem(method);
             });
         }
-        if (!!yclass.doc) {
-            ymlClass.setDocumentation(yclass.doc[0]);
-        }
+        ymlClass.setUserInformations(`(class) ${ymlClass.label}`, !!yclass.doc ? yclass.doc[0] : null);
```

Comment:
Kinda same thing here: maybe the details already show plainly enough that this is a class (but maybe not (I have yet to try it out)).

Reply:
`setUserInformations` sets both `detail` and `documentation`.